README.md

```diff
@@ -55,6 +55,15 @@ gradlew sonarQube -DsonarRunner.aemVersion=6.4
 
 Below you will find descriptions of all rules available in **AEM Rules for SonarQube** plugin.
 
+- **HTL-6** HTL automatically recognises the context for HTML output
```

Comment:
Could we merge this branch: https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/147/files#diff-04c6e90faac2675aa89e2176d2eec7d8
and move your descriptions to keep that convention.